src/Articalizer.jsx

```diff
@@ -0,0 +1,88 @@
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Button from "@mui/material/Button";
+import Card from "@mui/material/Card";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import CssBaseline from "@mui/material/CssBaseline";
+import Grid from "@mui/material/Grid";
+import Stack from "@mui/material/Stack";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+import Link from "@mui/material/Link";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+const defaultTheme = createTheme();
+
+export default function Articalizer(props) {
+  const { articles } = props;
+  const cards = [];
+  for (let i = 0; i < articles?.length; i++) {
+    cards.push(articles[i]);
```

Comment:
Just trying to figure out what this code is doing. You're pushing to an array called cards, but I can't see it being used anywhere?